### DISCUSSION TITLE
How to disable services notificaions?

### DISCUSSION THREAD

**Discussion by t1m013y** [winsiderss/systeminformer#2104] [Questions / Answers] [answered]
How to disable services creation/deletion notificaions?

**Comment by jxy-s** [comment, marked as answer]
To disable notifications:
Right-click tray icon > Notifications > ...
![image](https://github.com/winsiderss/systeminformer/assets/11687482/4ee3d011-0697-4c9c-a946-cd1221116127)

To filter notifications:
System > Options > Notifications - > ...
![image](https://github.com/winsiderss/systeminformer/assets/11687482/f5178722-4589-4629-8f2a-2a6c22d5a30c)

**Reply by t1m013y** [reply]
When I hover mouse on tray icon, this window pops up so I can't right click the tray icon.
![изображение](https://github.com/winsiderss/systeminformer/assets/91249047/cfeb89f7-e6e3-45dc-b89e-99b4d898e64d)

**Comment by Vasilich** [comment]
go to menu - view - tray icons - notifications and select/deselect all notifications you need:

![image](https://github.com/winsiderss/systeminformer/assets/1168434/90686601-7820-4b8d-a298-24e2d64be168)

**Reply by t1m013y** [reply]
I have no such option
![изображение](https://github.com/winsiderss/systeminformer/assets/91249047/d5579c01-5473-4241-9766-8e47321e306d)

**Comment by dmex** [comment]
@t1m013y 

The notification menu in Process Hacker can only be found when right-clicking the tray icon. You should probably update to the newer version: https://systeminformer.sourceforge.io/downloads

**Reply by t1m013y** [reply]
Thanks. Found newer portable version (i didn't know that process hacker renamed to system informer)
